engine/src/audio/audio_song.cpp

```diff
@@ -148,6 +148,7 @@ void AudioSong::work() {
 
   if (chunkReadStatus > 0) {
     WaitSema(fillbufferSema);  // wait until previous chunk wasn't finished
+    audsrv_wait_audio(chunkReadStatus);

```

Comment:
This change is been applied [here](https://github.com/h4570/tyra/pull/151/commits/200a68e2194054956416fa62913fa9cde15aa820), i will talk to wellinator about dropping this commit or i will close this one.